modules/util/EventEmitterForwarder.js

```diff
@@ -30,9 +31,25 @@ EventEmitterForwarder.prototype.forward = function(...args) {
     args[0] = this.dest;
 
     // Using bind.apply to pass the arguments as Array-like object ("arguments")
-    this.src.addListener(
+    this.unsubscribers.set(
```

Comment:
This is awesome to remove the listeners! Thanks for doing the effort. I'm wondering why we this functionality didn't exist.

But I don't think this whole thing will work in a generic way! Listenables will return the function for removing the listener and this would work but a normal EventEmitter will just return itself...

So maybe it will be better to store the actual listener that we attach in the unsubscribers Map and then use it call removeListener. This way this should work everything.

Reply:
Ok, I will do the change force push it in the commit. Is this ok?